neo/Prompt/Commands/Send.py

```diff
@@ -228,11 +228,17 @@ def process_transaction(wallet, contract_tx, scripthash_from=None, scripthash_ch
                                     change_address=scripthash_change,
                                     fee=fee,
                                     from_addr=scripthash_from)
+    except ValueError:
+        print("Insufficient funds. No unspent outputs available for building the transaction.\n"
+              "If you are trying to sent multiple transactions in 1 block, then make sure you have enough 'vouts'\n."
+              "Use `wallet unspent` and `wallet address split`, or wait until the first transaction is processed before sending another.")
+        return None
 
-        if tx is None:
-            logger.debug("insufficient funds")
-            return None
+    if tx is None:
+        logger.debug("insufficient funds")
```

Comment:
At this point, if there really is insufficient funds (i.e. the user tries to send more than the wallet balance), won't it not display, since the users doesn't usually see DEBUG info?

Reply:
nah, because real insufficient funds will also give the long message. Only if the wallet is not synced it will hit that, but then it has a `logger.warning` before that to indicate the wallet isn't synced.